gnovm/pkg/gnopkgfetch/gnopkgfetch.go

```diff
@@ -0,0 +1,130 @@
+package gnopkgfetch
```

Comment:
Can we please think of a better name? 🙏

Reply:
I made a proposal, let's switch conversation to this one: https://github.com/gnolang/gno/pull/3123/files/3d0533c0e2e072604f04088be861a571d38c7c2b#diff-ae08265dbde526e6779fdcaa901d52d2566769cc94b1b62cb6dcd3be54c368f2